### DISCUSSION TITLE
Intro: Madelaine Ley

### DISCUSSION THREAD

**Discussion by madelaineley** [life-itself/community#389] [Introductions]
Hi everyone! I'm happy to join the group of Life Itself researchers :) I'm a philosopher specializing in philosophy of technology, applied ethics, phenomenology and intersectional feminism. My work explores how tech shifts how we care for ourselves, one another and the more-than-human world. I'm beginning to learn more about spiritual ecology and how the senses invite ethical living. 

I'm making a big shift in my academic work and hoping my google scholar page will soon be more aligned with my spirit. For now, you can check out my substack, [Beauty in the Mire](https://madelaineley.substack.com), to see what I'm thinking about. I also did [a talk](https://www.youtube.com/watch?v=l6p0dKyFCmA) recently for Life Itself community group, which is a good example of how I mix philosophy, narrative and practice. 

Website: madelaineley.com

Looking forward to meeting you all! I'm traveling April 14th, so won't be able to meet you then.

**Comment by rufuspollock** [comment]
@madelaineley welcome Madelaine and thanks for sharing!
